cookbook/security/custom_authentication_provider.rst

```diff
@@ -433,27 +440,33 @@ to service ids that do not exist yet: ``wsse.security.authentication.provider``
 
     .. code-block:: php
 
-        // src/AppBundle/Resources/config/services.php
+        // app/config/services.php
         use Symfony\Component\DependencyInjection\Definition;
         use Symfony\Component\DependencyInjection\Reference;
 
-        $container->setDefinition('wsse.security.authentication.provider',
-            new Definition(
-                'AppBundle\Security\Authentication\Provider\WsseProvider', array(
-                    '',
-                    '%kernel.cache_dir%/security/nonces',
+        $container
+            ->setDefinition('wsse.security.authentication.provider',
+                new Definition(
+                    'AppBundle\Security\Authentication\Provider\WsseProvider', array(
+                        '', // User Provider
+                        '%kernel.cache_dir%/security/nonces',
+                    )
                 )
             )
-        );
-
-        $container->setDefinition('wsse.security.authentication.listener',
-            new Definition(
-                'AppBundle\Security\Firewall\WsseListener', array(
-                    new Reference('security.context'),
-                    new Reference('security.authentication.manager'),
+            ->setPublic(false)
+        ;
+
+        $container
+            ->setDefinition('wsse.security.authentication.listener',
+                new Definition(
```

Comment:
You used a line wrapping method here that doesn't comply with PSR. I would actually prefer if you saved the definition into a variable (`$definition`) first and then register that. An other way is to also put the service id on a new line